### PR TITLE
DOCSP-35785: added --deployment-name flag and page metadata

### PR DESCRIPTION
**Pull request by osharaf-mdb**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-35785

- [App Services CLI](https://preview-mongodbosharafmdb.gatsbyjs.io/atlas-app-services/DOCSP-35785-add-deployment-name-flag/cli/): added updated CLI docs

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [N/A] Did you tag pages appropriately?
  - genre
  - programming_language
  - meta.keywords
  - meta.description
- [x] Describe your PR's changes in the Release Notes section
- [N/A] Create a Jira ticket for related docs-realm work, if any

### Release Notes

- **Reference**
  - App Services CLI: Regenerate and add updated documentation pages for App Services CLI and all children pages.

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/cli/appservices-push.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ appservices push
    :depth: 1
    :class: singlecol
 
-Imports and deploys changes from your local directory to your app (alias: import)
+Pushes and deploys changes from your local directory to your app
```
Comment (lindseymoore) [inline source/cli/appservices-push.txt:15]:
```suggestion
Imports and deploys changes from your local directory to your app (alias: import)
```
A little bit confusing, but the push command does technically 'import' local changes to your App

File: source/cli/appservices-push.txt [hunk 2]
```diff
@@ -44,23 +44,23 @@ Options
    * - --local
      - string
      - false
-     - Specify the local filepath of an app to be imported
+     - Specify the local filepath of an app to be pushed
```
Comment (lindseymoore) [inline source/cli/appservices-push.txt:47]:
```suggestion
     - Specify the local filepath of an app to be imported
```
Just double checking, were these wording changes from push to import made in the new set or did you make them? We normally accept the description as they come from engineering

**Review by lindseymoore — COMMENTED** [review]
Double checking on the wording change: is 'push' the new term over 'import' in the regenerated file? If so, let me know, and I'll approve!

File: source/cli/appservices-push.txt (hunk 2 above)
Comment (osharaf-mdb) [reply source/cli/appservices-push.txt:47]: These were changed from the generated doc, I did not make them manually!

File: source/cli/appservices-apps-init.txt
Comment (osharaf-mdb) [inline source/cli/appservices-apps-init.txt:1]: I'm not entirely sure about teh format/syntax of the NOTE

File: source/cli/appservices-apps-create.txt
Comment (osharaf-mdb) [inline source/cli/appservices-apps-create.txt:1]: I'm not entirely sure about the format/syntax of the NOTE

File: source/cli/appservices-apps-create.txt [hunk 3]
```diff
@@ -55,19 +55,19 @@ Options
    * - -n, --name
      - string
      - false
-     - Name your new app
+     - Name your new app (Note: This name will be used internally and cannot be changed later. Names must only include: ASCII letters, numbers, _ -)
```
Comment (lindseymoore) [inline source/cli/appservices-apps-create.txt:58]:
```suggestion
     - Name your new app (Note: This name will be used internally and cannot be changed later. Names must only include: ASCII letters, numbers, underscores ( _ ), or dashes ( - ))
```
clarity nit

File: source/cli/appservices-apps-init.txt [hunk 4]
```diff
@@ -51,19 +51,19 @@ Options
    * - -n, --name
      - string
      - false
-     - Name your new app
+     - Name your new app (Note: This name will be used internally and cannot be changed later. Names must only include: ASCII letters, numbers, _ -)
```
Comment (lindseymoore) [inline source/cli/appservices-apps-init.txt:54]:
```suggestion
     - Name your new app (Note: This name will be used internally and cannot be changed later. Names must only include: ASCII letters, numbers, underscores ( _ ), or dashes ( - ))
```

File: source/cli/appservices-function.txt [hunk 5]
```diff
@@ -69,7 +69,7 @@ Inherited Options
 Related Commands
 ----------------
 
-* :ref:`appservices-function-list` - List the Functions defined for your app
+* :ref:`appservices-function-list` - List the functions of your app
```
Comment (lindseymoore) [inline source/cli/appservices-function.txt:72]:
```suggestion
* :ref:`appservices-function-list` - List the Functions defined for your app
```
clarity nit

File: source/cli/appservices-schema-validate.txt [hunk 6]
```diff
@@ -82,7 +82,7 @@ Options
    * - --local
      - string
      - false
-     - Specify the local filepath of an app services project (Note: The local filepath must be an absolute path or the command will fail)
+     - Specify the local filepath of an app services project
```
Comment (lindseymoore) [inline source/cli/appservices-schema-validate.txt:85]:
```suggestion
     - Specify the local filepath of an app services project (Note: The local filepath must be an absolute path or the command will fail)
```
Please keep the note; I added it manually.

File: source/cli/appservices-users-count.txt [hunk 7]
```diff
@@ -12,7 +12,7 @@ appservices users count
    :depth: 1
    :class: singlecol
 
-Displays the number of total application users of your app, across all Auth Provider types.
+Displays the total count of application users of your app.
```
Comment (lindseymoore) [inline source/cli/appservices-users-count.txt:15]:
```suggestion
Displays the number of total application users of your app, across all Auth Provider types.
```
I would keep the first description for clarity!

File: source/cli/appservices-users-delete.txt [hunk 8]
```diff
@@ -53,15 +53,15 @@ Options
...
    * - --state
      - string
      - false
-     - Filter the Realm app's users by state (Default value: <none>; Allowed values: enabled, disabled)
+     - Filter the App Services app's users by state (Default value: <none>; Allowed values: enabled, disabled)
    * - --provider
      - Set
      - false
-     - Filter the Realm app's users by provider type (Default value: <none>; Allowed values: <none>, "local-userpass", "api-key", "oauth2-facebook", "oauth2-google", "anon-user", "custom-token", "oauth2-apple", "custom-function") This value defaults to [].
+     - Filter the App Services app's users by provider type (Default value: <none>; Allowed values: <none>, "local-userpass", "api-key", "oauth2-facebook", "oauth2-google", "anon-user", "custom-token", "oauth2-apple", "custom-function") This value defaults to [].
```
Comment (lindseymoore) [inline source/cli/appservices-users-delete.txt:64]:
```suggestion
     - Filter the App Services app's users by provider type (Default value: <none>; Allowed values: <none>, "local-userpass", "api-key", "oauth2-facebook", "oauth2-google", "anon-user", "custom-token", "oauth2-apple", "custom-function"). This value defaults to [].
```
period

File: source/cli/appservices-users-list.txt [hunk 9]
```diff
@@ -51,19 +53,19 @@ Options
...
    * - --provider
      - Set
      - false
-     - Filter the Realm app's users by provider type (Default value: <none>; Allowed values: <none>, "local-userpass", "api-key", "oauth2-facebook", "oauth2-google", "anon-user", "custom-token", "oauth2-apple", "custom-function") This value defaults to [].
+     - Filter the App Services app's users by provider type (Default value: <none>; Allowed values: <none>, "local-userpass", "api-key", "oauth2-facebook", "oauth2-google", "anon-user", "custom-token", "oauth2-apple", "custom-function") This value defaults to [].
    * - -l, --limit
      - string
      - false
-     - Specify the number of users to list. A max of 50 users can be retured in one call. 
+     - Specify the number of users to list
```
Comment (lindseymoore) [inline source/cli/appservices-users-list.txt:68]:
```suggestion
     - Specify the number of users to list. A max of 50 users can be retured in one call. 
```
Keep the first!

File: source/cli/appservices-users-list.txt [hunk 10]
```diff
@@ -51,19 +53,19 @@ Options
...
    * - --state
      - string
      - false
-     - Filter the Realm app's users by state (Default value: <none>; Allowed values: enabled, disabled)
+     - Filter the App Services app's users by state (Default value: <none>; Allowed values: enabled, disabled)
    * - --provider
      - Set
      - false
-     - Filter the Realm app's users by provider type (Default value: <none>; Allowed values: <none>, "local-userpass", "api-key", "oauth2-facebook", "oauth2-google", "anon-user", "custom-token", "oauth2-apple", "custom-function") This value defaults to [].
+     - Filter the App Services app's users by provider type (Default value: <none>; Allowed values: <none>, "local-userpass", "api-key", "oauth2-facebook", "oauth2-google", "anon-user", "custom-token", "oauth2-apple", "custom-function") This value defaults to [].
```
Comment (lindseymoore) [inline source/cli/appservices-users-list.txt:64]:
```suggestion
     - Filter the App Services app's users by provider type (Default value: <none>; Allowed values: <none>, "local-userpass", "api-key", "oauth2-facebook", "oauth2-google", "anon-user", "custom-token", "oauth2-apple", "custom-function"). This value defaults to [].
```

File: source/cli/appservices-users-revoke.txt [hunk 11]
```diff
@@ -53,15 +53,15 @@ Options
...
    * - --state
      - string
      - false
-     - Filter the Realm app's users by state (Default value: <none>; Allowed values: enabled, disabled)
+     - Filter the App Services app's users by state (Default value: <none>; Allowed values: enabled, disabled)
    * - --provider
      - Set
      - false
-     - Filter the Realm app's users by provider type (Default value: <none>; Allowed values: <none>, "local-userpass", "api-key", "oauth2-facebook", "oauth2-google", "anon-user", "custom-token", "oauth2-apple", "custom-function") This value defaults to [].
+     - Filter the App Services app's users by provider type (Default value: <none>; Allowed values: <none>, "local-userpass", "api-key", "oauth2-facebook", "oauth2-google", "anon-user", "custom-token", "oauth2-apple", "custom-function") This value defaults to [].
```
Comment (lindseymoore) [inline source/cli/appservices-users-revoke.txt:64]:
```suggestion
     - Filter the App Services app's users by provider type (Default value: <none>; Allowed values: <none>, "local-userpass", "api-key", "oauth2-facebook", "oauth2-google", "anon-user", "custom-token", "oauth2-apple", "custom-function"). This value defaults to [].
```

**Review by lindseymoore — APPROVED** [review]
LGTM, after fixing the small comments! Please let me know if you have any questions! Thanks for making these updates :)

**Pull request merged**